ice.c

```diff
@@ -3049,14 +3129,14 @@ void *janus_ice_send_thread(void *data) {
 					JANUS_LOG(LOG_HUGE, "[%"SCNu64"] Fixing SSRCs (local %u, peer %u)\n", handle->handle_id,
 						video ? stream->video_ssrc : stream->audio_ssrc,
 						video ? stream->video_ssrc_peer : stream->audio_ssrc_peer);
-					janus_rtcp_fix_ssrc((char *)&sbuf, pkt->length, 1,
+					janus_rtcp_fix_ssrc(NULL, (char *)&sbuf, pkt->length, 1,
 						video ? stream->video_ssrc : stream->audio_ssrc,
 						video ? stream->video_ssrc_peer : stream->audio_ssrc_peer);
 				} else {
 					/* Plan B involved, we trust the plugin to set the right 'local' SSRC and we don't mess with it */
 					JANUS_LOG(LOG_HUGE, "[%"SCNu64"] Fixing peer SSRC (Plan B, peer %u)\n", handle->handle_id,
 						video ? stream->video_ssrc_peer : stream->audio_ssrc_peer);
-					janus_rtcp_fix_ssrc((char *)&sbuf, pkt->length, 1, 0,
+					janus_rtcp_fix_ssrc(NULL, (char *)&sbuf, pkt->length, 1, 0,
```

Comment:
you don't need to cast any of these buffers to `(char *)`. just `sbuf` should do it.

(this is actually one of those interesting cases where the address value of `sbuf` is the same as the address value of `&sbuf`. the types are a bit different. but just `sbuf` will work for a `char *`)